lib/views/signin_page/signin_page.dart

```diff
@@ -1,19 +1,130 @@
+                Align(
+                  alignment: Alignment.topRight,
+                  child: GestureDetector(
+                    onTap: () {
+                      Navigator.maybePop(context);
+                    },
+                    child: Container(
+                      height: 26,
+                      width: 26,
+                      decoration: BoxDecoration(
+                        border: Border.all(
+                          color: AppColors.grey6,
+                          width: 3,
+                        ),
+                        color: AppColors.lightYellow,
+                        shape: BoxShape.circle,
+                      ),
+                      child: const Icon(
+                        Icons.close,
+                        size: 20,
+                      ),
+                    ),
+                  ),
+                ),
+                Container(
+                  height: 84,
+                  width: 84,
+                  decoration: BoxDecoration(
+                    border: Border.all(
+                      color: AppColors.yellowPrimary,
+                      width: 3,
+                    ),
+                    color: AppColors.lightYellow,
+                    shape: BoxShape.circle,
+                  ),
+                  child: const Center(
+                    child: Text(
+                      '🤔',
+                      style: TextStyle(
+                        color: AppColors.grey6,
+                        fontSize: 36,
+                        fontWeight: FontWeight.w500,
+                      ),
+                    ),
+                  ),
+                ),
+                const Gap(16),
+                Column(
+                  children: const [
+                    Text(
+                      'This email has not been registered yet',
+                      style: TextStyle(
+                        color: AppColors.grey0,
+                        fontSize: 24,
+                        height: 30 / 24,
+                        fontWeight: FontWeight.w500,
+                      ),
+                      textAlign: TextAlign.center,
+                    ),
+                    Gap(8),
+                    Text(
+                      'Visit Devfest.com to register your email address',
+                      style: TextStyle(
+                        color: AppColors.grey6,
+                        fontSize: 16,
+                        height: 22 / 16,
+                        fontWeight: FontWeight.w500,
+                      ),
+                      textAlign: TextAlign.center,
+                    ),
+                  ],
+                ),
+                const Gap(24),
+                const DevFestButton(
+                  text: 'Register Email Address',
+                ),
+              ],
+            ),
+          ),
+        );
+      },
+    );
+  }
+
   @override
   Widget build(BuildContext context) {
+    var vm = ref.read(signinVM);
```

Comment:
Don't access the VM directly for functions.
use `ref.read(vm.select((it)=>  it.whatYouWant));`